static/js/fit.js

```diff
@@ -981,24 +981,12 @@ $( document ).ready(function() {
       }
   });
 
-  delegate('#text-segment', 'submit', (e,t) => {
-    let mysave = $('#fit_wysiwyg_editor').html();
-    $('#text_segment_content').val(mysave);
-  });
-
-  delegate('#lesson-edit-form', 'submit', (e,t) => {
-    let mysave = $('#fit_wysiwyg_editor').html();
-    $('#further_reading').val(mysave);
-  });
-
-  delegate('#course-edit-form', 'submit', (e,t) => {
-    let mysave = $('#fit_wysiwyg_editor').html();
-    $('#course_summary').val(mysave);
-  });
-
-  delegate('#add-edit-resource', 'submit', (e,t) => {
-    let mysave = $('#fit_wysiwyg_resource').html();
-    $('#resource_description').val(mysave);
+  delegate('[data-fit-wysiwyg]', 'submit', (e,t) => {
+    let p = t.closest('[data-fit-wysiwyg]');
+    let preview = p.querySelector('[data-fit-wysiwyg-preview]');
+    let testarea = p.querySelector('[data-fit-wysiwyg-textarea]');
```

Comment:
You should change this variable to `textarea` to avoid confusion. 😄

Reply:
😅